doc/performance-investigations/0013-gunicorn-worker-thread-config/summary.md

```diff
@@ -0,0 +1,112 @@
+# Gunicorn worker and thread count configuration
+
+Questionnaire runner currently uses the formula `(2 x $num_cores) + 1` when configuring the number of Gunicorn workers and threads. This results in 7 workers by 7 threads for our current vCPU count of 3.
+This document highlights the results from running the application with different numbers of workers and thread permutations. Both workers and threads ranged from 1 to 10. 
+
+## Benchmark profile
+
+| Option                             | Value                        |
+|------------------------------------|------------------------------|
+| Requests file                      | census_household_gb_eng.json |
+| Run time                           | 20m                          |
+| User wait time minimum             | 1                            |
+| User wait time maximum (exclusive) | 3                            |
+| Clients                            | 225                          |
+| Hatch rate                         | 225                          |
+| Number of workers                  | 1 to 10                      |
+| Number of threads                  | 1 to 10                      |
+| Questionnaire Runner version       | v3.56.0                      |
+| Questionnaire Runner min/min pods  | 3                            |
+| Max nodes per zone                 | 1                            |
```

Comment:
This is to highlight that 3 pods were between 3 nodes ensuring the kubesystem pods had to be one of the 3 nodes to ensure consistent between runs.